examples/it.xsemantics.example.fj/src/it/xsemantics/example/fj/typing/fj-separated.xsemantics

```diff
@@ -33,6 +33,17 @@ import org.eclipse.xtext.EcoreUtil2
  */
 system it.xsemantics.example.fj.typing.FjSepTypeSystem extends FjTypeSystem
 
+copyright
```

Comment:
Thanks, I take care of this. I made changes to the copyright tag which was previously introduced by @JensN4 and this tag wasn't updated yet.